app/src/main/java/org/shadowice/flocke/andotp/Activities/AuthenticateActivity.java

```diff
@@ -56,22 +55,24 @@
 import org.shadowice.flocke.andotp.Tasks.AuthenticationTask;
 import org.shadowice.flocke.andotp.Tasks.AuthenticationTask.Result;
 import org.shadowice.flocke.andotp.Utilities.Constants;
+import org.shadowice.flocke.andotp.View.AutoFillable.AutoFillableTextInputEditText;
 
 import static org.shadowice.flocke.andotp.Utilities.Constants.AuthMethod;
 
 public class AuthenticateActivity extends BaseActivity
         implements EditText.OnEditorActionListener, View.OnClickListener {
+    private final AutoFillableTextInputEditText.AutoFillTextListener autoFillTextListener = text -> checkPassword(text.toString());
```

Comment:
I missed this when fixing the merge conflicts, this needs to use `startAuthTask(text.toString()` now instead if `checkPassword(text.toString())`.

Reply:
Done 👍